src/nav/style-guide.yml

```diff
@@ -153,6 +153,8 @@ pages:
                 path: /docs/style-guide/writing-docs/article-templates/prometheus-template
               - title: Troubleshooting docs guide
                 path: /docs/style-guide/writing-docs/article-templates/troubleshooting-docs-guide
+              - title: UI installation routines
```

Comment:
The file path seems a little odd here. I think the path should reflect its location in the nav

Reply:
Shawn, I'll move this to writer-workflow.